pkg/relayer/config/types.go

```diff
@@ -0,0 +1,132 @@
+package config
+
+import "net/url"
+
+// YAMLRelayMinerConfig is the structure used to unmarshal the RelayMiner config file
+// TODO_DOCUMENT(@red-0ne): Add proper README documentation for yaml config files.
+type YAMLRelayMinerConfig struct {
+	SigningKeyName string                         `yaml:"signing_key_name"`
+	SmtStorePath   string                         `yaml:"smt_store_path"`
+	Pocket         YAMLRelayMinerPocketConfig     `yaml:"pocket"`
```

Comment:
Not a fan of the word `Pocket` since it's too generic/overloaded.

How about `Node` or PocketNode`?